pages/common/ppmtv.md

```diff
@@ -0,0 +1,17 @@
+# ppmtv
+
+> Make a PPM Image look like taken from an american TV.
+> Dim every other row of image data down by the specified dim factor.
+> More information: <https://netpbm.sourceforge.net/doc/ppmtv.html>.
+
+- Give the PPM image an American TV appearance:
+
+`ppmtv dimfactor {{path/to/file.ppm}}`
```

Comment:
Maybe mention whether or not this changes the image in-place. In case the change is not in-place, maybe change to 
```suggestion
- Give the PPM image an American TV appearance:

`ppmtv {{dimfactor}} {{path/to/file.ppm}} > {{path/to/output.ppm}}`
```

Reply:
`>` redirection will only work if the image is printed to `stdout`, and I don't think that's the case.

Reply:
> `>` redirection will only work if the image is printed to `stdout`, and I don't think that's the case.

It is the case for many other tools in Netbpm, that's why I'm asking 😅

Reply:
I just tried it on my computer. Indeed, the program prints to stdout.